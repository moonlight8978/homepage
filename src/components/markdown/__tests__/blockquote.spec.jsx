import React from 'react'
import { render } from '@testing-library/react'

import Blockquote from '../blockquote'

test('renders string', () => {
  const children = 'Hello'
  const { container } = render(<Blockquote>{children}</Blockquote>)
  expect(container.firstChild).toMatchSnapshot()
})

test('renders react node', () => {
  const children = <>Hello</>
  const { container } = render(<Blockquote>{children}</Blockquote>)
  expect(container.firstChild).toMatchSnapshot()
})
