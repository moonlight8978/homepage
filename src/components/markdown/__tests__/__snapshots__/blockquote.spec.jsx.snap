// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders react node 1`] = `
<blockquote
  class="blockquote"
  data-testid="markdown-blockquote"
>
  Hello
</blockquote>
`;

exports[`renders string 1`] = `
<blockquote
  class="blockquote"
  data-testid="markdown-blockquote"
>
  Hello
</blockquote>
`;
